rkt/run.go

```diff
@@ -457,14 +461,14 @@ func (e *envFileMap) Set(s string) error {
 			continue
 		}
 		pair := strings.SplitN(scanner.Text(), "=", 2)
```

Comment:
It's basically bikeshedding, but I think this would be easier to read if we split out a `line = scanner.Text()` and skipped based on whether that `line` looked like a comment before splitting. It better fits what a comment is so it increases readability.